src/browse.vue

```diff
@@ -509,6 +515,25 @@ export default {
 
             return params;
         },
+
+        processTableFields(endpointFields) {
+            const newFields = [];
+            this.extraFields.forEach((fieldDefinition, index) => {
+                // find field to replace the render
+                const fieldIndex = endpointFields.findIndex(field => [fieldDefinition.name, fieldDefinition.field].includes(field.name))
+                if (fieldIndex != -1) {
+                    const fieldName = endpointFields[fieldIndex].name;
+                    endpointFields[fieldIndex] = { ...endpointFields[fieldIndex], ...fieldDefinition, fieldName };
+                } else {
+                    newFields.push(index)
+                }
+            })
+            newFields.forEach((fieldIndex) => {
+                endpointFields.push(this.extraFields[fieldIndex]);
+            })
```

Comment:
Why iterate here again and not do this push into `endpointFields` in line 528?

Also:
```suggestion
            newFields.forEach((fieldIndex) => {
                endpointFields.push(this.extraFields[fieldIndex]);
            });
```